src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -317,25 +319,38 @@ private static void forNonComplexVariants(final JavaRDD<AssemblyContigWithFineTu
             throws IOException {
         final SimpleNovelAdjacencyAndChimericAlignmentEvidence simpleNovelAdjacencyAndChimericAlignmentEvidence = pair._1;
         final List<SvType> svTypes = pair._2;
```

Comment:
I think you're assuming here that there are either 1 or 2 elements of this list. Can you add a validation of that?